### ISSUE TITLE
Suggestion Change of the Supply-Roll with modifier

### ISSUE THREAD

**Issue by Moradion** [pwatson100/alienrpg#243]
Could there be an option to add a modifier (right-click) to the supply roll switches? Similar to skills. After some talents modify them.

**Comment by pwatson100** [comment]
Good idea.  It will be in the next release.  As per the rules all supply rolls are capped at 6 dice so you can never roll more that 6 dice.
